reporter/src/main/java/org/arquillian/cube/q/recorder/TakeNetworkChaosInformation.java

```diff
@@ -80,32 +80,32 @@ public void addToxicInfoToToxics(QNetworkChaosToxic.ToxicAction toxicAction, Tox
 
     public void reportToxicConfiguration(@Observes After event, ReporterConfiguration reporterConfiguration) throws IOException {
 
-        final String fileName = event.getTestMethod().getName() + ".json";
+        final Method testMethod = event.getTestMethod();
+        final String testMethodName = testMethod.getName();
+        final String fileName = testMethodName + ".json";
 
         final FileEntry fileEntry = createFileEntryWithJSON(reporterConfiguration, fileName);
+        Reporter.createReport(new TestMethodReport(testMethodName))
+                .addEntry(fileEntry)
```

Comment:
at this point, I would expect addKeyValueEntry where the key is some kind of description what the file contains etc... As a value you can use any Entry, so also FileEntry